allocations/allocations_api.py

```diff
@@ -0,0 +1,31 @@
+import requests
+import traceback
+from django.conf import settings
+import logging
+from keystoneauth1.identity import v3
+from keystoneauth1 import session
+from project.view import get_ks_auth_and_session
+
+logger = logging.getLogger('allocations')
+
+class BalanceServiceClient:
+
+    def call( self, project_code ):
+        auth, sess = get_ks_auth_and_session()
+        headers = auth.get_headers(sess)
+        headers['Accept'] = 'application/json'
+        
+        url = self.make_url( project_code )
+        logger.info('Requesting %s from allocations balance service ...', url)
```

Comment:
I know I am very guilty of writing log messages like this, saying "I'm _about_ to do something", but I don't think it's the right way to log for this type of thing. Logging based on intent is a bit dangerous because if you're just looking at the logs, you have no idea if the code _actually_ did what was intended. Instead, I recommend doing this style:

```python
logger.info('Successfully retrieved balance for project %s', project_code)
logger.exception('Failed to retrieve balance for project %s', project_code)
```

If you want to include debug information, you can do it on another verbosity level:

```python
logger.debug('Response from %s: %s', url, data)
```

This keeps INFO logs clean (which they should be) - INFO should be telling you important operational information about what the system is doing. DEBUG should be for when you need to figure out why something is _not_ working properly. And of course ERROR is for when things fail.